src/blog/jwt-identification-and-hot-streams/index.md

```diff
@@ -0,0 +1,173 @@
+
+What can AnyCable do for these users? Let's take a look at how Turbo Streams work with Rails.
+
+We have the [turbo-rails][] gem, which integrate Action Cable with Hotwire. All you need is to drop a helper into your template to start consuming streams:
+
+```erb
+<%= turbo_stream_from some_model %>
+```
+
+The helper adds a [`<turbo-cable-stream-source>` element][turbo-cable-stream-source], which initiates an Action Cable subscription. The corresponding server-side code is as follows:
+
+```ruby
+class Turbo::StreamsChannel < ActionCable::Channel::Base
+  extend Turbo::Streams::Broadcasts, Turbo::Streams::StreamName
+
+  def subscribed
+    if verified_stream_name = self.class.verified_stream_name(params[:signed_stream_name])
+      stream_from verified_stream_name
+    else
+      reject
+    end
+  end
+end
+```
+
+The `#verified_stream_name` method uses an [`ActiveSupport::MessageVerifier`][message-verifier] to decode and verify the stream name. The subscription is rejected if the signed stream name is invalid.
+
+Structurally speaking, the code above is the same as the example we wrote for JWT identification. What does this mean? It means that, **once again**, we can "move" the implementation to an AnyCable Go server and **create subscriptions without touching RPC**.
+
+This is exactly what we did as a part of the [signed streams][signed-streams-docs] feature. Further, we did it not only for Turbo Streams, but for Cable Ready as well (since its [`#stream_from` implementation][cr-stream-from] is nearly the same).
+
+> By combining JWT identification with signed streams, it's possible to completely avoid running an RPC server, in case you only need to use Hotwire or CableReady functionality.
+
+Below is a visualization of RPC server metrics during Hotwire benchmarks with four different configurations (denoted by blue markers), from left to right: baseline (AnyCable PRO without any features enabled), with JWT identification, with signed streams, and with both features enabled at the same time.
+
+<figure class="blog--figure">
+  <img class="blog--media" title="AnyCable metrics when running with and without JWT and 'hot' streams" src="./grafana.png" width="1364" height="624">
+</figure>
+
+It shouldn't come as any surprise that the last one shows zeros for RPC metrics—it wasn't impacted at all. We can also note that CPU usage with AnyCable Go is the lowest in the fourth scenario—**dealing with tokens and signatures is "cheaper" than performing gRPC calls**.
+
+<div class="divider"></div>
+
+We've added these features in response to our users needs, and because we believe that we're able to build a much better product by working together. Want to join the team? Give [AnyCable a try today](/)!
```

Comment:
QUESTION: Are we talking about paid features in this article? (The PRO version is not free, correct?)

Reply:
I'm not an experienced pitch-writer, so, I'd prefer to leave it up to you 🙂; WDYT?

Reply:
**Here's my suggestion, at least, this can be a starting point:**
We've added these features in response to your needs. We believe that, thanks to user feedback, we're able to build a much better product together. Who knows? Maybe your request will be the next feature we add! Ready to get started? Give [AnyCable a try today](/)!

Reply:
```suggestion
We've added these features in response to your needs. We believe that, thanks to user feedback, we're able to build a much better product together. Who knows? Maybe your request will be the next feature we add! Ready to get started? Give [AnyCable a try today](/)!
```